config/environments/test.rb

```diff
@@ -48,6 +48,7 @@
   # The :test delivery method accumulates sent emails in the
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
+  config.action_mailer.default_url_options = {host: "example"}
```

Comment:
let's read this from constants file. we can add another variable there called `HOST`. also should this not be example.com instead of just example?